main.html

```diff
@@ -69,118 +69,7 @@ <h3 class="intro-header2">I’am a Software Developer</h3>
-                    <li class="tag">Ruby</li>
-                </ul>
-            </div>
-            <a class="btn btn-2 see-project-btn" href="#">See Project</a>
-        </div>
-        <div class="card4 card">
-            <div class="content">
-                <h3>Profesional Art Printing Data</h3>                
-                <p class="p-format">
-                    A daily selection of privately personalized reads; no accounts or sign-ups required. 
-                    has been the industry's standard
-                </p>
-                <ul class="tag-group">
-                    <li class="tag">html</li>
-                    <li class="tag">bootstrap</li>
-                    <li class="tag">Ruby</li>
-                </ul>
-            </div>
-            <a class="btn btn-2 see-project-btn" href="#">See Project</a>
-        </div>
-        <div class="card5 card">
-            <div class="content">
-                <h3>Profesional Art Printing Data</h3>                
-                <p class="p-format">
-                    A daily selection of privately personalized reads; no accounts or sign-ups required. 
-                    has been the industry's standard
-                </p>
-                <ul class="tag-group">
-                    <li class="tag">html</li>
-                    <li class="tag">bootstrap</li>
-                    <li class="tag">Ruby</li>
-                </ul>
-            </div>
-            <a class="btn btn-2 see-project-btn" href="#">See Project</a>
-        </div>
-        <div class="card6 card">
-            <div class="content">
-                <h3>Profesional Art Printing Data</h3>                
-                <p class="p-format">
-                    A daily selection of privately personalized reads; no accounts or sign-ups required. 
-                    has been the industry's standard
-                </p>
-                <ul class="tag-group">
-                    <li class="tag">html</li>
-                    <li class="tag">bootstrap</li>
-                    <li class="tag">Ruby</li>
-                </ul>
-            </div>
-            <a class="btn btn-2 see-project-btn" href="#">See Project</a>
-        </div>
-        <div class="card7 card">
-            <div class="content">
-                <h3>Profesional Art Printing Data</h3>                
-                <p class="p-format">
-                    A daily selection of privately personalized reads; no accounts or sign-ups required. 
-                    has been the industry's standard
-                </p>
-                <ul class="tag-group">
-                    <li class="tag">html</li>
-                    <li class="tag">bootstrap</li>
-                    <li class="tag">Ruby</li>
-                </ul>
-            </div>
-            <a class="btn btn-2 see-project-btn" href="#">See Project</a>
-        </div>
+        <!-- This Section was built dinamically in javaScript  -->
```

Comment:
Wonderfull work 🤓 The projects section is created dynamically with the information from the projects array as expected  🔥